src/Microsoft.Sbom.Api/SBOMValidator.cs

```diff
@@ -45,7 +45,7 @@ public async Task<bool> ValidateSbomAsync()
         return isSuccess;
     }
 
-    public async Task<bool> ValidateSbomAsync(
+    public async Task<IList<EntityError>> ValidateSbomAsync(
```

Comment:
It would be better if you could return a new object that has a property for validation result (true/false) and the list of Errors